File: 프로그래머스/Level.1/30. 이상한 문자 만들기.js
function solution(s) {
  return s
    .split(' ')
    .map((word) =>
      word
        .split('')
        .map((char, idx) => (idx % 2 === 0 ? char.toUpperCase() : char))
        .join('')
    )
    .join(' ');
}

solution('try hello world');

// 하나 이상의 공백문자
// 짝수번째 알파벳은 대문자
// 홀수번째 알파벳은 소문자

File: 프로그래머스/Level.1/30. 이상한 문자 만들기.py
def solution(s):
    words = s.split(' ')
    new_list = []
    for i in words:
        str =''
        for j in range(len(i)):
            str+=i[j].upper() if j%2==0 else i[j].lower()
        new_list.append(str)
    return ' '.join(new_list)